### PR TITLE
change

### PR DESCRIPTION
**Pull request by neko039**
增加student_id欄位到experance.txt

### CODE REVIEW

File: backend/server.py [hunk 1]
```diff
@@ -95,18 +95,6 @@ def signup():
...
-        if account == "":
-            return json.dumps({"error_message": "account can't empty."}), 500
-        try:
-            with open("static/accounts.txt") as lines:
-                for line in lines:
-                    json_data = json.loads(line)
-                    if json_data["account"] == account:
-                        return json.dumps({"error_message": "account is exist."}), 400
-        except:
-            return json.dumps({"error_message": "can't load account data."}), 500
```
Comment (yillkid) [inline backend/server.py:109]: 參考 [git amend](https://backlog.com/git-tutorial/tw/stepup/stepup7_1.html) 修改你的提交。

File: backend/server.py [hunk 2]
```diff
@@ -233,4 +233,4 @@ def backend_account_manage():
     return redirect(url_for("index"))
 
 if __name__ == "__main__":
-    app.run(debug = True, threaded = True, host = "0.0.0.0", port = 5000)
```
Comment (yillkid) [inline backend/server.py:236]: specific port number to 5001

File: backend/static/experience.txt [hunk 3]
```diff
@@ -4,3 +4,6 @@
 {"student_id": "10000001", "activity_kind": "other", "experience": "234"}
 {"student_id": "8903001", "activity_kind": "lecture", "experience": "123"}
 {"student_id": "8903001", "activity_kind": "lecture", "experience": "111"}
+{"activity_kind": "lecture", "experience": "sfsdfg"}
+{"activity_kind": "art", "experience": "test 2222", "student_id": "isu"}
+{"activity_kind": "lecture", "experience": "test 2224\r\n", "student_id": "isu"}
```
Comment (1step2god) [inline backend/static/experience.txt:9]: "test 2224\r\n" is wried.

**Comment by 1step2god** [comment]
Don't use "$ git add ." now, use "$ git add <file>".Until I configure git.ignore

**Comment by neko039** [comment]
image
git amend 不能使用

**Comment by 1step2god** [comment]
It didn’t work.

**Pull request closed without merging**